esi_leap/tests/resource_objects/test_ironic_node.py

```diff
@@ -197,3 +210,11 @@ def test_get_node_cache(self, mock_gn):
         self.assertEqual(fake_get_node,
                          test_ironic_node._get_node())
         mock_gn.assert_not_called
+
+    @mock.patch('esi_leap.common.ironic.get_node')
+    def test_get_unknown_node(self, mock_gn):
+        unknown_get_node = UnknownIronicNode()
+        mock_gn.return_value = None
+        test_unknown_node = ironic_node.IronicNode(fake_uuid)
+        test_unknown_node._node = unknown_get_node
```

Comment:
You'll want to test that this is equal to resource_objects.ironic_node.UnknownIronicNode